Add test for Plugin::CPANFile

dboehmer opened the pull request:
I had the idea of adding a `comment` parameter to `Plugin::CPANFile` and started by writing a test for the existing module. Here is the test for the current state of the plugin.

Just after finishing the original idea I realized there is https://github.com/rjbs/Dist-Zilla/pull/629 which is nearly equivalent to my changes but has a reasonable default comment. For my implementation see
https://github.com/dboehmer/Dist-Zilla/tree/plugin-cpanfile-comments

You might want to merge this PR before https://github.com/rjbs/Dist-Zilla/pull/629 and add add a change to the test to that PR. I volunteer to deliver that patch to #629.

Grinnz commented:
The blead failure is not related to your code (see the failure on master).

rjbs commented:
@dboehmer I have returned after a rather lengthy wandering in the wilderness.  I have applied #629, and am not quite sure whether this set of changes is still applicable.  I realize it has been quite a while.  I will probably close this PR if I don't hear back in the near future.

Thank you for your work on behalf of DZ!

rjbs commented:
Nevermind, I see!  I have amended the test and applied it.  Thanks!

dboehmer commented:
Wow, that feature was nearer than I expected;-) I wasn't that fast to reply.

> I have amended the test and applied it. Thanks!

Are you sure you pushed all the changes? I can't see any changes in `t/` in https://github.com/rjbs/Dist-Zilla/commit/9fd9a4d7d5d1d42f9c22b83ea88c0ae220f7f02b

dboehmer commented:
Just in case I rebased and adjusted my test to the merged Plugin::CPANFile and force-pushed to [my branch `test-plugin-cpanfile`](https://github.com/dboehmer/Dist-Zilla/tree/test-plugin-cpanfile)

You might want to merge that. Do you want a new PR for this?

rjbs commented:
Nah, thanks, I've applied your changes to master, and then tweaked them to match rjbs style.
